src/views/document/RouteView.vue

```diff
@@ -186,6 +188,25 @@ export default {
 
       return result;
     },
+    documentType() { // is-editable mixin needs this property
+        return this.$documentUtils.getDocumentType(this.document.type);
+      },
+    PromoteHistory() {
```

Comment:
do not use upper case for function names.
It's a test function, so better to be named e.g. `shouldPromoteHistory`